src/types/Champion.ts

```diff
@@ -80,6 +81,24 @@ export interface ChampionPricing {
   sale: number;
 }
 
+/**
+ * The sprite data of a champion.
+ */
+export interface ChampionSprite {
```

Comment:
I made the import in `src/types/index.ts` since there was no dependencies in `src/types/Champion.ts` in 2196cf00b87af1e4e234529c9d0a0ef2a9fd579a. Please let me know if you explicitly want the interfaces imported/exported through `Champion.ts`. u.u

Reply:
I meant moving this interface to `src/types/ChampionSprite.ts` instead of keeping it in `src/types/Champion.ts`

Reply:
Yeah that is what I did in the commit I mentioned. Am I misunderstanding something or did you want other changes made?

Reply:
AHhh, mb, I was looking at older code not updated one.